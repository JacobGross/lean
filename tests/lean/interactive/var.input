VISIT foo.lean
SYNC 3
import data.prod
open prod
definition foo A := λB, A × B
WAIT
INFO 3