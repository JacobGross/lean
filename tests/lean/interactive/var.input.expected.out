-- BEGINWAIT
-- ENDWAIT
-- BEGININFO
-- TYPE|3|15
Type
-- ACK
-- IDENTIFIER|3|15
A
-- ACK
-- SYMBOL|3|20
λ
-- ACK
-- TYPE|3|21
Type
-- ACK
-- IDENTIFIER|3|21
B
-- ACK
-- TYPE|3|24
Type
-- ACK
-- IDENTIFIER|3|24
A
-- ACK
-- TYPE|3|26
Type → Type → Type
-- ACK
-- SYMBOL|3|26
×
-- ACK
-- TYPE|3|28
Type
-- ACK
-- IDENTIFIER|3|28
B
-- ACK
-- ENDINFO
